nomad/local/integration-tests.nomad

```diff
@@ -317,7 +323,9 @@ job "integration-tests" {
 
       env {
         # aws vars
-        AWS_REGION                  = var.aws_region
+        AWS_REGION = var.aws_region
+        # TODO: Do the "USE_LOCALHOST_SENTINEL_VALUE" check here in case we
```

Comment:
I think this comment belongs at the top in the locals section, but absolutely agree